From e865a93000913b4597607289356114cd159f4e28 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Fri, 21 May 2021 03:02:29 +0000
Subject: [PATCH] fix compile error for cross compile

On some distros, such as fedora32, cross compile failed with following
error since host library is used. undefined reference to
`stat64@GLIBC_2.33'

According doc of ld, set searchdir begins with "=", but not hardcoded
locations.

Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/libgtop/-/merge_requests/26]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 472f44b..ed6a4d7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -270,8 +270,8 @@ AC_ARG_ENABLE(fatal-warnings,
                         [Define to enable fatal warnings]))
 
 dnl These definitions are expanded in make.
-LIBGTOP_LIBS='-L$(libdir)'
-LIBGTOP_INCS='-I$(includedir)/libgtop-2.0'
+LIBGTOP_LIBS='-L=$(libdir)'
+LIBGTOP_INCS='-I=$(includedir)/libgtop-2.0'
 
 if test x$libgtop_have_sysinfo = xyes ; then
   LIBGTOP_INCS="$LIBGTOP_INCS -DHAVE_LIBGTOP_SYSINFO"
-- 
2.26.2

